README.md

```diff
@@ -0,0 +1,72 @@
+
+
+### apt_sources_ubuntu
+| Option Key                  | Type            |  Default/Required [¹](#__required)   | Description                                                                                                         |
+|-----------------------------|-----------------|:------------------------------------:|---------------------------------------------------------------------------------------------------------------------|
+| `release`                   | string          | `{{ ansible_distribution_release }}` | Release string to set                                                                                               |
+| `mirror`                    | string          |    `http://de.archive.ubuntu.com/`   | Mirror to use for apt entries (The leading `/` is required)                                                         |
+| `enable_sources`            | boolean         |                `False`               | Enable `deb-src` entries                                                                                            |
+| `components`                | list of strings |        `['main', 'universe']`        | Components to enable (valid values are: `main`, `restricted`, `universe`, `multiverse`)                             |
+| `security_repo`             | boolean         |                `True`                | Enable security repository                                                                                          |
+| `update_repo`               | boolean         |                `True`                | Enable general update repository                                                                                    |
+| `backports`                 | boolean         |                `False`               | Enable backports repository                                                                                         |
+| `proposed`                  | boolean         |                `False`               | Enable proposed repository                                                                                          |
+| `canonical`                 | boolean         |                `False`               | Enable ubuntu partner (canonical) repository                                                                        |
+| `canonical_mirror`          | string          |    `http://archive.canonical.com/`   | Ubuntu canonical mirror to use                                                                                      |
+| `additional_source_strings` | list of strings |                 `[]`                 | Additional apt sources to add (each string is additional line in `/etc/apt/sources.list` without any modifications) |
+
+<a id="__required">¹</a> Variable is not required unless no default is given or other specified
```

Comment:
Is it necessary to mention this? Because all variables have a default value.

Reply:
I think it's more clear to mention this here, this helps also to improve this role later.